servers/src/main/java/tachyon/master/lineage/journal/LineageEntry.java

```diff
@@ -42,6 +43,23 @@
   private final String mJobOutputPath;
   private final long mCreationTimeMs;
 
+  @JsonCreator
+  public LineageEntry(@JsonProperty("id") long id,
```

Comment:
Could you add a javadoc notifying callers that this method is for Json serialization only?